lib/src/platform_mq.dart

```diff
@@ -50,7 +50,7 @@ class PlatformMQImpl implements PlatformMQ {
       _client.connectionMessage = MqttConnectMessage()
         ..startClean()
         ..withWillMessage(lastWillMessage)
-        ..withWillQos(MqttQos.atMostOnce)
+        ..withWillQos(MqttQos.atLeastOnce)
```

Comment:
We want to ensure that the last will message is delivered at least once so a queued service request will be cancelled.